daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -831,64 +868,69 @@ private[lf] object Speedy {
   }
 
   private[speedy] final case class KOverApp(
+      savedBase: Int,
       newArgs: Array[SExprAtomic],
       frame: Frame,
-      actuals: Actuals,
-      envSize: Int)
+      actuals: Actuals)
       extends Kont
       with SomeArrayEquals {
     def execute(vfun: SValue, machine: Machine) = {
-      machine.restoreEnv(frame, actuals, envSize)
+      machine.restoreBase(savedBase);
```

Comment:
Semicolons at the end of instructions? Are you getting ready for Rust? 😉

Reply:
No idea what happened here :shrug:  Removed all the trailing semis which I added.